tests/parsers/test_pw.py

```diff
@@ -379,6 +379,31 @@ def test_pw_npools_too_high(fixture_localhost, generate_calc_job_node, generate_
     assert calcfunction.exit_status == node.process_class.exit_codes.ERROR_NPOOLS_TOO_HIGH.status
 
 
+def test_tot_magnetization(
+    fixture_localhost, generate_calc_job_node, generate_parser, generate_inputs, data_regression
+):
+    """Test the parsing of a calculation that specifies tot_magnetization.
+
+    In this case there are two Fermi energies, see:
+
+    https://lists.quantum-espresso.org/pipermail/users/2011-April/020089.html
+    """
+    name = 'tot_magnetization'
+    entry_point_calc_job = 'quantumespresso.pw'
+    entry_point_parser = 'quantumespresso.pw'
+
+    node = generate_calc_job_node(entry_point_calc_job, fixture_localhost, name, generate_inputs())
+    parser = generate_parser(entry_point_parser)
+    results, calcfunction = parser.parse_from_node(node, store_provenance=False)
+
+    assert calcfunction.is_finished, calcfunction.exception
+    assert calcfunction.is_finished_ok, calcfunction.exit_status
+    assert 'output_parameters' in results
+    data_regression.check({
+        'output_parameters': results['output_parameters'].get_dict(),
+    })
```

Comment:
Why not just check for the presence of the new keys? It _is_ possible that this change could break other code, but given that it adds a single elif clause, that is unlikely. Don't think it is necessary to check the entire output parameters node

Reply:
Makes sense! It's been adjusted.